specification/cognitiveservices/data-plane/LUIS/Runtime/stable/v3.0/LUIS-Runtime.json

```diff
@@ -5,7 +5,7 @@
     "version": "3.0"
   },
   "x-ms-parameterized-host": {
-    "hostTemplate": "{Endpoint}/luis/v3.0",
+    "hostTemplate": "{Endpoint}/luis/prediction/v3.0",
```

Comment:
It's stable version of API. I don't think we can allow changes in host template without API version change..